python/lsst/ts/wep/task/generateDonutFromRefitWcsTask.py

```diff
@@ -216,14 +216,12 @@ def formatDonutCatalog(self, catalog):
         sourceSchema = afwTable.SourceTable.makeMinimalSchema()
         measBase.SingleFrameMeasurementTask(schema=sourceSchema)  # expand the schema
         # add coord_raErr,  coord_decErr to the schema
```

Comment:
Extra whitespace after comma. Also, can you just add a little info on why we do this now.